.typos.toml

```diff
@@ -23,7 +23,7 @@ extend-ignore-identifiers-re = []
 extend-ignore-re = [
     "(?i)aks",
     "ANDed",
-    "PT_REGS_PARM3",
+    "PT_REGS_PARM",
```

Comment:
Is this change intend it?

Reply:
Yes, the linter started to complain about `PT_REGS_PARM4`. I figured that `PT_REGS_PARM` is always ok. :)